.eslintrc

```diff
@@ -6,6 +6,16 @@
     "jsx-a11y/anchor-is-valid": [ "error", {
       "components": [ "Link" ],
       "specialLink": [ "to" ]
+    }],
+    "jsx-a11y/label-has-associated-control": [ "error", {
+      "required": {
+        "some": [ "nesting", "id"  ]
+      }
+    }],
+    "jsx-a11y/label-has-for": [ "error", {
```

Comment:
``` this rule is been deprecated. i think its not needed ```

Reply:
Removed that rule 7d10e9764d00d82f3419b8961d8677e8798d97f9